animations/effects.json

```diff
@@ -324,5 +324,73 @@
 				}
 			]
 		}
+	],
+	"effect:spell-effect-shields-of-the-spirit-security": [
+		{
+			"trigger": "effect",
+			"preset": "onToken",
+			"file": "jb2a.markers.shield_rampart.complete.01.orange",
+			"options": {
+				"scaleToObject": {
+					"value": 1.5
+				},
+				"tieToDocuments": true,
+				"mask": false
+			},
+			"contents": [
+				{
+					"predicate": [
+						"jb2a:free"
+					],
+					"file": "jb2a.markers.shield_rampart.complete.01.orange",
+					"filter": {
+						"type": "ColorMatrix",
+						"options": {
+							"hue": 150
+						}
+					}
+				},
```

Comment:
This doesn't work like that. This section will never happen due to not having a `trigger`.

Also, the file is exactly the same? What is different about this?

Reply:
Ah I'll fix the trigger sorry, and this is the only rampart usable for the non premium users.

I was under the misunderstanding that the individual parts stacked.

I think I'll also redo it to take into account what aura type you have (holy, unholy etc.)
But I'll update my PR tonight probably

Reply:
Only whatever is in the `contents` stack on top of one another. Otherwise they are separate.